github_connector/models/github_repository_branch.py

```diff
@@ -292,7 +295,10 @@ def _compute_organization_serie_id(self):
 
     @api.depends("complete_name")
     def _compute_local_path(self):
-        source_path = tools.config.get("source_code_local_path", "")
+        source_path = (
+            tools.config.get("source_code_local_path", "")
+            or os.environ["SOURCE_CODE_LOCAL_PATH"]
```

Comment:
This one should have also more priority than config one, and this way it's going to fail if not present. Shouldn't it be?:

```suggestion
            or os.environ.get("SOURCE_CODE_LOCAL_PATH")
```